utils/src/test/kotlin/ExtensionsTest.kt

```diff
@@ -45,6 +45,42 @@ class ExtensionsTest : WordSpec({
         }
     }
 
+    "File.isSymbolicLink" should {
+        val tempDir = createTempDir()
+        val file = tempDir.resolve("file").apply { createNewFile() }
+        val directory = tempDir.resolve("directory").apply { safeMkdirs() }
+
+        "not detect files as symbolic links on Windows".config(enabled = Os.isWindows) {
```

Comment:
Consider starting test names with: "return false for..."  (symbolic links on windows)

Consider also turning "symbolic links" into "symbolic file links" and the below one into "symbolic directory links"

Reply:
I've done the former but not the latter.